packages/server/src/models/flower.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { UserShelfActions } from '../constants/user-shelf';
+
+@Entity()
+export class Flower {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  shelfId: number;
```

Comment:
"shelfId" from Flower table must be a foreign key for "id" in Shelf table. Also we need this relations for all related ids.
https://wanago.io/2019/01/21/typescript-express-tutorial-8-types-of-relationships-with-postgres-and-typeorm/

Reply:
Must it be? We haven't been strictly using foreign keys before now, was there a reason?

Reply:
No reason. But it good to have them for excluding cases when ids invalid or not exist

Reply:
Fixed

Reply:
Fixed